### DISCUSSION TITLE
Is it possible to edit polygon's side number ?

### DISCUSSION THREAD

**Discussion by Gatarlo** [matteobruni/tsparticles#1722] [Q&A] [answered]
Hi,

I just had a small question, is it possible to edit the number of side when the shape is set on polygon ?

Thank you ! :)

**Comment by matteobruni** [comment, marked as answer]
Hi @Gatarlo,

Yes it's possible, you can see how in this sample: https://codepen.io/matteobruni/pen/WNxPpMZ

**Reply by Gatarlo** [reply]
Perfect thank you ! I was writing "nb_side" instead of "sides"
